fs/littlefs/Kconfig

```diff
@@ -6,17 +6,81 @@ config FS_LITTLEFS
 		Build the LITTLEFS file system. https://github.com/littlefs-project/littlefs.
 
 if FS_LITTLEFS
+config FS_LITTLEFS_ADVANCED_BLOCK_DEVICE_CFG
```

Comment:
let's remove FS_LITTLEFS_ADVANCED_BLOCK_DEVICE_CFG and extend the config option directly

Reply:
Would this cover your suggestions?
```
  fs->cfg.read_size      = fs->geo.blocksize *
                           CONFIG_FS_LITTLEFS_READ_SIZE_FACTOR;
  fs->cfg.prog_size      = fs->geo.blocksize *
                           CONFIG_FS_LITTLEFS_PROGRAM_SIZE_FACTOR;
  fs->cfg.block_size     = fs->geo.erasesize *
                           CONFIG_FS_LITTLEFS_BLOCK_SIZE_FACTOR;
  fs->cfg.block_count    = fs->geo.neraseblocks / fs->cfg.block_size;
  fs->cfg.block_cycles   = CONFIG_FS_LITTLEFS_BLOCK_CYCLE;
  fs->cfg.cache_size     = fs->geo.blocksize *
                           CONFIG_FS_LITTLEFS_CACHE_SIZE_FACTOR;
#if (CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE == 0)
  fs->cfg.lookahead_size = lfs_min(lfs_alignup(fs->cfg.block_count, 64) / 8,
                                   fs->cfg.read_size);
#else
  fs->cfg.lookahead_size = CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE;
#endif
```

Reply:
Yes, but block_count need change like this:
```
fs->cfg.block_count    = fs->geo.neraseblocks / CONFIG_FS_LITTLEFS_BLOCK_SIZE_FACTOR;
```